src/main/resources/edu/cornell/kfs/module/purap/cu-spring-purap.xml

```diff
@@ -277,4 +277,13 @@
 
 	<bean id="eInvoiceDao" parent="platformAwareDao"
 		  class="edu.cornell.kfs.module.purap.dataaccess.impl.CuEinvoiceDaoOjb"/>
+
+	<bean id="jaggaerPermissionTypeService"  parent="kimTypeService" class="edu.cornell.kfs.module.purap.identity.JaggaerPermissionTypeService" />
+	<bean id="jaggaerRoleTypeService" parent="kimTypeService" class="edu.cornell.kfs.module.purap.identity.JaggaerRoleTypeService" />
```

Comment:
For these permission type and role type beans, you may want to change their parents to "kimPermissionTypeService" and "kimRoleTypeService", respectively.

Reply:
ok